### PR TITLE
Fix contributing guide build

### PR DESCRIPTION
**Pull request by cwharris**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ conda activate srf
 ```
 #### Build SRF
 ```bash
-mkdir $SRF_HOME/build
+mkdir SRF_HOME/build
```
Comment (mdemoret-nv) [inline CONTRIBUTING.md:64]: Why did you remove the `$`? And leave it in the next line?

Comment (cwharris) [reply CONTRIBUTING.md]: I was moving to fast and changed the wrong line. Fixed.

**Review by mdemoret-nv — APPROVED** [review]

**Comment by mdemoret-nv** [comment]
@gpucibot merge
